EmbeddedScripts.sln

```diff
@@ -3,10 +3,72 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
 VisualStudioVersion = 16.0.31409.214
 MinimumVisualStudioVersion = 10.0.40219.1
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Tests", "Tests", "{DB199BCF-967A-46EE-89E5-BBC0B6D59BE5}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EmbeddedScripts.Shared.Tests", "Tests\EmbeddedScripts.Shared.Tests\EmbeddedScripts.Shared.Tests.csproj", "{8F5F5F05-27E6-403A-BB07-DC4B0B90AEDD}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EmbeddedScripts.CSharp.Shared.Tests", "Tests\EmbeddedScripts.CSharp.Shared.Tests\EmbeddedScripts.CSharp.Shared.Tests.csproj", "{3024EE77-31C5-4152-B2E2-7804F4F52BF1}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EmbeddedScripts.CSharp.Roslyn.Tests", "Tests\EmbeddedScripts.CSharp.Roslyn.Tests\EmbeddedScripts.CSharp.Roslyn.Tests.csproj", "{A60F0AFA-706B-4631-9088-6F2C1AB15EE1}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EmbeddedScripts.CSharp.Roslyn.Scripting.Tests", "Tests\EmbeddedScripts.CSharp.Roslyn.Scripting.Tests\EmbeddedScripts.CSharp.Roslyn.Scripting.Tests.csproj", "{DBA6B831-D9A6-4B6A-8371-9C08D873363B}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EmbeddedScripts.Shared", "Src\EmbeddedScripts.Shared\EmbeddedScripts.Shared.csproj", "{FE3F6392-3653-411A-A2C5-08D381F9E047}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EmbeddedScripts.CSharp.Shared", "Src\EmbeddedScripts.CSharp.Shared\EmbeddedScripts.CSharp.Shared.csproj", "{0AA360F7-BB3B-46C9-9739-3A7B4F8414C8}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EmbeddedScripts.CSharp.Roslyn.Scripting", "Src\EmbeddedScripts.CSharp.Roslyn.Scripting\EmbeddedScripts.CSharp.Roslyn.Scripting.csproj", "{FEBC4CAC-CAC4-4867-AC03-957C267A3A38}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EmbeddedScripts.CSharp.Roslyn", "Src\EmbeddedScripts.CSharp.Roslyn\EmbeddedScripts.CSharp.Roslyn.csproj", "{825E6C37-4DF7-452B-AF17-63A35093953B}"
```

Comment:
let's rename project CSharp.Roslyn to something else like CSharp.Roslyn.Compilation?
otherwise it looks like CSharp.Roslyn.Scripting is dependent on CSharp.Roslyn but it's not